api/XPathExpression.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/XPathExpression",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "29"
```

Comment:
The data added in https://github.com/mdn/browser-compat-data/pull/7644 says that `XPathEvaluator.createExpression()` was supported since Chrome 1, so this doesn't seem right. Maybe the interface object wasn't exposed?

Reply:
That seems the most likely -- I'll reopen this with custom tests introduced in 1.1.7!